stylesheets/CSS/typography.css

```diff
@@ -0,0 +1,55 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Satisfy&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:wght@700&family=Satisfy&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:wght@700&family=Raleway&family=Satisfy&family=Work+Sans:wght@300&display=swap');
+/* font-family: 'Raleway', sans-serif;
```

Comment:
Avoid giving comments like this.

Reply:
okay..will take care from now on... i gave it for my reference while coding them. Thank you for feedback